vboxlauncher@mockturtl/files/vboxlauncher@mockturtl/applet.js

```diff
@@ -237,12 +245,17 @@ MyApplet.prototype = {
     }
     this.addUpdater()
   }
-  
+
 ,  startVboxImage: function(id) {
     let cmd = this.vboxMajorVersion() >= 6 ? CMD_VBOX6_VM : CMD_VBOX_VM
     Util.spawnCommandLine(cmd + id)
   }
-  
+
+,  startVboxImageHeadless: function(id) {
+    let cmd = this.vboxMajorVersion() >= 6 ? CMD_VBOX6_VM : CMD_VBOX_VM
```

Comment:
As noted above, the git history indicates vboxmanage <= 5 takes `--startvm` versus `startvm`.

Can we confirm it vbox <= 5 does work with `--type headless`?

Reply:
Maybe someone else can confirm but acording to this tutorial, there is vboxmanage at least from 5.2
https://websiteforstudents.com/installing-virtualbox-5-2-ubuntu-17-04-17-10/

Reply:
Okay, I don't quite follow the sea of commands and symlinks in 5.x, but I see `--type headless` is supported.

```sh
 $ vboxmanage --help
Oracle VM VirtualBox Command Line Management Interface Version 5.2.24_Debian
(C) 2005-2019 Oracle Corporation
All rights reserved.
 ...
  startvm                   <uuid|vmname>...
                            [--type gui|sdl|headless|separate]
                            [-E|--putenv <NAME>[=<VALUE>]]
...
```

For reference, v6.0:

```sh
$ virtualbox --help
Oracle VM VirtualBox VM Selector v6.0.4
(C) 2005-2019 Oracle Corporation
All rights reserved.

No special options.

If you are looking for --startvm and related options, you need to use VirtualBoxVM.
$ vboxmanage startvm
Usage:

VBoxManage startvm          <uuid|vmname>...
                            [--type gui|sdl|headless|separate]
                            [-E|--putenv <NAME>[=<VALUE>]]
```